packages/grafana-ui/src/components/TimePicker/TimeZonePicker/TimeZoneOption.tsx

```diff
@@ -104,34 +102,26 @@ const getStyles = stylesFactory((theme: GrafanaTheme) => {
       flex-shrink: 0;
       white-space: nowrap;
       cursor: pointer;
-      border-left: 2px solid transparent;
       padding: 6px 8px 4px;
 
       &:hover {
-        background: ${theme.colors.dropdownOptionHoverBg};
+        background: ${theme.colors.action.hover};
 
-        span.${offsetClassName} {
-          background: ${offsetHoverBg};
-        }
+        // span.${offsetClassName} {
+        //   background: ${theme.colors.action.hover};
+        // }
```

Comment:
nit: should this be removed?

Reply:
oh, yea, was going to test firs how it looked and if this was needed, forgot about it :)